src/tools/remove_surface.cc

```diff
@@ -52,8 +52,10 @@ struct remove_surface_tool : tool
         if (!can_use(rc))
             return;
 
+        block *bl = rc->block;
         auto index = normal_to_surface_index(rc);
-        auto mesh = asset_man.get_surface_mesh(index);
+
+        auto mesh = asset_man.get_surface_mesh(index, bl->surfs[index]);
```

Comment:
What's the path forward here for variants? We pass the location to get_surface_mesh as well?